add docs for ALTER SYSTEM (#190)
cedardb/docs: content/references/configuration.md

A value set via an environment variable takes precedence over the value defined in the configuration file.
{{% /callout %}}

### ALTER SYSTEM

You can also update the configuration file from a SQL session with the [`ALTER SYSTEM`](/docs/references/sessions/altersystem)
statement, without editing the file by hand:

```sql
ALTER SYSTEM SET buffersize = '8G';
```

> As with the config file, these changes take effect the next time CedarDB starts.

Reviewer: ```suggestion
As with the config file, these changes only take effect after a restart.
```
